.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="USE_PROJECT_PROFILE" value="false" />
```

Comment:
The option name and value should be placed on separate lines for better readability.